cypress/integration/sales/void_a_sales_invoice_then_invoice_the_billing_candidates_again.js

```diff
@@ -339,8 +339,20 @@ describe('Void Sales Invoice and invoice the billing candidates again', function
     cy.getStringFieldValue('M_Product_ID').should('contain', productName);
     cy.getStringFieldValue('QtyEntered').should('equal', originalQuantity.toString(10));
     cy.getStringFieldValue('QtyOrdered').should('equal', originalQuantity.toString(10));
-    cy.getStringFieldValue('QtyDelivered ').should('equal', qtyDelivered);
-    cy.getStringFieldValue('QtyInvoiced ').should('equal', qtyInvoiced);
+
+    // backend is flaky and keeps switching between '' and '0' and back to '', so we need this workaround
```

Comment:
@TheBestPessimist do you think it makes sense to create a backend task to fix that ? Having types problems on top of what we already have makes it even harder to maintain the whole thing :)